<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerwindow</class>
 <widget class="QWidget" name="registerwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>0</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="mainHorLayout"/>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>531</width>
     <height>411</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);
font: 11pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>账号名称:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="acountLineEdit">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>0</y>
      <width>202</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="pswLineEdit">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>50</y>
      <width>202</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>124</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);
font: 11pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>登录密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="repeatPswLineEdit">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>110</y>
      <width>202</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>124</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);
font: 11pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>确认密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="verCodeLineEdit">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>180</y>
      <width>202</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>180</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);
font: 11pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>验证码:</string>
    </property>
   </widget>
   <widget class="QLabel" name="verCodeLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="registerPushBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);
font: 11pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeVerCodePushBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);
font: 11pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>换一张</string>
    </property>
   </widget>
   <widget class="QLabel" name="hintAccLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hintPswLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hintRepeatLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hintVerCodeLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
